### DISCUSSION TITLE
LAB 05.01 Error en la autenticación

### DISCUSSION THREAD

**Discussion by davidlon13** [rramosp/ai4eng.v1#190] [00 - General]
Buenas Noches.

Quería preguntar sobre un error que me esta a apareciendo cuando quiero autenticarme para entregar los lab.

![image](https://user-images.githubusercontent.com/108371648/185771301-5b7a3958-94a8-420c-b94a-c3cf13d7a55f.png)
 
Hace pocos días que esta sucediendo, no se si es problema de mi cuenta o es algo general.

Saludos y gracias.

**Comment by rramosp** [comment]
gracias david ... cual es el error en concreto que te sale?

On Sat, Aug 20, 2022 at 8:18 PM davidlon13 ***@***.***> wrote:

> Buenas Noches.
>
> Quería preguntar sobre un error que me esta a apareciendo cuando quiero
> autenticarme para entregar los lab.
>
> [image: image]
> <https://user-images.githubusercontent.com/108371648/185771301-5b7a3958-94a8-420c-b94a-c3cf13d7a55f.png>
>
> Hace pocos días que esta sucediendo, no se si es problema de mi cuenta o
> es algo general.
>
> Saludos y gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/190>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIN5O56U6BCSO5Q6R5DV2F7WDANCNFSM57ECVMXQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by davidlon13** [reply]
Profe, exactamente cuando ejecuto la celda que debería abrir los cuadros para poner usuario y contraseña, me aparece una ventana extra del Colab que no permite autenticarse como lo hacíamos habitualmente.

**Comment by rramosp** [comment]
ya veo ... parece que los de google volvieron a cambiar algo en la
estructure de los notebooks en colab.

prueba a hacer esto, en una celda después de ejecutar siempre la primera
(wget, init, etc.):

student = session.Session(init.endpoint).login(course_id=init.course_id,
lab_id="L02.04")

y pones el lab que corresponde ... ¿puedes hacer una submisión normalmente?

On Sun, Aug 21, 2022 at 6:15 PM davidlon13 ***@***.***> wrote:

> Profe, exactamente cuando ejecuto la celda que debería abrir los cuadros
> para poner usuario y contraseña, me aparece una ventana extra del Colab que
> no permite autenticarse como lo hacíamos habitualmente.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/190#discussioncomment-3442869>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZILK5B5OUXV33WCD7LDV2K2CZANCNFSM57ECVMXQ>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by davidlon13** [reply]
Si profe muchas gracias, ya me funcionó.
